.github/workflows/spellchecker.yml

```diff
@@ -0,0 +1,12 @@
+name: Spell checking
+on: [pull_request]
+
+jobs:
+  typos:
+    name: Spell Check with Typos
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout Actions Repository
+      uses: actions/checkout@v4
+    - name: typos-action
+      uses: crate-ci/typos@v1.21.0
```

Comment:
Can you add newline, to the end of the file?

https://stackoverflow.com/a/729795